src/status_im/contexts/centralized_metrics/core.cljs

```diff
@@ -0,0 +1,88 @@
+(ns status-im.contexts.centralized-metrics.core
```

Comment:
There are a few conventions we follow which can be applied to this namespace.

- Events go in `events.cljs` file. We stopped using `core` namespaces to hold events, effects and so on.
- Effects go in `effects.cljs` file
- Effects are prefixed with `:effects*`, like `:effects.centralized-metrics/toggle-metrics`